### ISSUE TITLE
Feature request: filter to add icons

### ISSUE THREAD

**Issue by xxsimoxx** [khacoder/canuck-cp#1]
It would be nice to add custom icons using a filter.

[This line](https://github.com/khacoder/canuck-cp/blob/b210c9e177adaf3038e8f122f05f5aca38ce34a9/includes/icons.php#L188) can be changed with
`return apply_filters('canuckcp_icons', $canuckcp_icons);`
and [this one](https://github.com/khacoder/canuck-cp/blob/b210c9e177adaf3038e8f122f05f5aca38ce34a9/includes/icons.php#L334) with 
`return apply_filters('canuckcp_icon_select', $canuckcp_icon_select);`

With this filter it's super easy to add more FontAwesome icons or customer's logo for example.
Here the snippet to add icons (to be put into an utility plugin):
```php
add_filter ( 'canuckcp_icons',            'xxsimoxx_add_icons' ); 
add_filter ( 'canuckcp_icon_select',  'xxsimoxx_icon_select' ); 

function xxsimoxx_add_icons( $icons ) {
	$icons += [
		'paw' => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!-- Font Awesome Free 5.15.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) --><path d="M256 224c-79.41 0-192 122.76-192 200.25 0 34.9 26.81 55.75 71.74 55.75 48.84 0 81.09-25.08 120.26-25.08 39.51 0 71.85 25.08 120.26 25.08 44.93 0 71.74-20.85 71.74-55.75C448 346.76 335.41 224 256 224zm-147.28-12.61c-10.4-34.65-42.44-57.09-71.56-50.13-29.12 6.96-44.29 40.69-33.89 75.34 10.4 34.65 42.44 57.09 71.56 50.13 29.12-6.96 44.29-40.69 33.89-75.34zm84.72-20.78c30.94-8.14 46.42-49.94 34.58-93.36s-46.52-72.01-77.46-63.87-46.42 49.94-34.58 93.36c11.84 43.42 46.53 72.02 77.46 63.87zm281.39-29.34c-29.12-6.96-61.15 15.48-71.56 50.13-10.4 34.65 4.77 68.38 33.89 75.34 29.12 6.96 61.15-15.48 71.56-50.13 10.4-34.65-4.77-68.38-33.89-75.34zm-156.27 29.34c30.94 8.14 65.62-20.45 77.46-63.87 11.84-43.42-3.64-85.21-34.58-93.36s-65.62 20.45-77.46 63.87c-11.84 43.42 3.64 85.22 34.58 93.36z"/></svg>',
	];
	return $icons;
}
	
function xxsimoxx_icon_select($icons) {
	$icons += [
		'paw' => 'paw',
	];
	return $icons;
}
```

**Comment by khacoder** [comment]
I've never used filters much, so it's not in my nature to think about how they can help me. Great suggestion, I will look at it for the next update.

**Comment by khacoder** [comment]
REsolved in 1.0.5
